### ISSUE TITLE
Installing and running sonar scanner runs into HTTP 401 

### ISSUE THREAD

**Issue by groie** [bellingard/sonar-scanner-npm#51]
**Steps to reproduce**
1 . Delete sonar-scanner binary (e.g. ```rm -rf /home/hic/.sonar/native-sonar-scanner/sonar-scanner-3.0.3.778-linux/bin/sonar-scanner```)
2. install newest sonarqube-scanner ```npm install --global sonarqube-scanner@2.1.1```
3. run it ```sonar-scanner```

This yields following error:

```
[10:19:42] Starting SonarQube analysis...
[10:19:42] Checking if executable exists: /home/hic/.sonar/native-sonar-scanner/sonar-scanner-3.2.0.1227-linux/bin/sonar-scanner
[10:19:42] Could not find executable in "/home/hic/.sonar/native-sonar-scanner".
[10:19:42] Proceed with download of the platform binaries for SonarQube Scanner...
[10:19:42] Creating /home/hic/.sonar/native-sonar-scanner
[10:19:42] Downloading from https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-3.2.0.1227-linux.zip
[10:19:42] (executable will be saved in cache folder: /home/hic/.sonar/native-sonar-scanner)
[10:19:43] ERROR: impossible to download and extract binary: Response code 401 (Unauthorized)
[10:19:43]        SonarQube Scanner binaries probably don't exist for your OS (linux).
[10:19:43]        In such situation, the best solution is to install the standard SonarQube Scanner (requires a JVM).
[10:19:43]        Check it out at https://redirect.sonarsource.com/doc/install-configure-scanner.html
```

**Comment by groie** [comment]
I am not blocked with a workaround 

```export SONAR_SCANNER_MIRROR=https://dl.bintray.com/sonarsource/SonarQube/org/sonarsource/scanner/cli/sonar-scanner-cli/3.2.0.1227/```

However I suspect the default install should work as well.

**Comment by bellingard** [comment]
This is fixed in 2.1.2 that I'm going to release in a couple of minutes.
